### DISCUSSION TITLE
v1.0.2

### DISCUSSION THREAD

**Discussion by rafael-figueiredo-alves** [rafael-figueiredo-alves/eFirebase#14] [General]
- Fixed a unit that was missing in a uses clause

<hr /><em>This discussion was created from the release <a href='https://github.com/rafael-figueiredo-alves/eFirebase/releases/tag/v1.0.2'>v1.0.2</a>.</em>

**Comment by Nhatsave** [comment]
Obrigado, a parte reclamada antes esta resolvida, mas notei um erro  no eFirebase.Interfaces haver com um END nao esperado depois do comentario sobre Firestore,

removendo o comentario todo sobre firestore o codigo avanca e detecta outros erros no arquivo eFirebase.Responses.Auth
envolvendo (TJSONValue, ParseJSONValue,  DisposeOf)  acredito que sao situacoes funcionais no Delphi mas que nao sao tratadas da mesma forma no Lazarus. Se tiver o lazarus instalado posso mandar o demo do meu projecto, acho que facilitara a depuracao do codigo.

**Reply by Nhatsave** [reply]
Aqui esta meu projecto
[Uploading project1_Firebase.rar…]()

**Comment by rafael-figueiredo-alves** [comment]
Ernesto, eu acabei de instalar o Lazarus e estarei analisando todos os erros. Muito obrigado pelas informações.

**Comment by Nhatsave** [comment]
Obrigado eu, acredito que sua biblioteca será de muita valia, hoje em dia
quase todos querem ter um servidor como firebase, e com a sua biblioteca
poderá se fazer uso mesmo no Lazarus oquê antes não era possível. Vou dar
feedback sempre que possível e recomendar alguns colegas da faculdade, um
vídeo também pode ajudá-lo a expandir... Aquele abraço

**Comment by rafael-figueiredo-alves** [comment]
@Nhatsave 
Eu já estou na fase final de correção da Lib. Estou executando testes e vou desenvolver um pequeno app de exemplo em Lazarus para a Lib. Em breve lhe aviso dos reparos, OK?

**Comment by Nhatsave** [comment]
Está certo, agradeço bastante

**Comment by rafael-figueiredo-alves** [comment]
@Nhatsave 
Ernesto, enfim terminei a implementação dos ajustes para que a lib do eFirebase funcione corretamente no Lazarus. Eu realizei testes e montei uma aplicação Demo em Lazarus que vc pode usar para verificar como funciona a implementação da LIB. Para usar, precisei instalar a biblioteca Indy no Lazarus e no menu Projeto, acesse inspetor do projeto e adicione referência ao IndyLaz (Indy). Realize seus testes e qualquer dúvida, fique a vontade para postar aqui ou, melhor, abrir uma issue. 

A versão 1.0.5 (#17 ) traz toda a correção necessária. 

Fico no aguardo de seu feedback/retorno.

**Comment by Nhatsave** [comment]
Meu muito obrigado, e darei feedback assim que possível
